server/rest/submission.py

```diff
@@ -128,6 +131,22 @@ def postSubmission(self, phase, folder, params):
             self.model('phase', 'challenge').requireAccess(
                 phase, user, level=AccessType.WRITE)
 
+        # Site admins may override the submission creation date
+        created = None
+        if 'date' in params:
+            if not user.get('admin'):
```

Comment:
I pushed a commit to use `self.requireAdmin()` with a message, but it's not showing up on the web--perhaps that's related to the service outage a few minutes ago.